tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -430,6 +430,23 @@ def config_bridge_domain(agent, test_bd, stepinfo='bridge-domain config:')
   end
 end
 
+# Helper for setting system bridge-domain configs
+def config_sys_bd(agent, test_sys_bd, stepinfo='sys-bd config:')
+  step stepinfo do
+    # NOTE: This should convert to using puppet resource, however, the cli
+    # does not allow changes to bridge-domain without removing existing BD's,
+    # which means we are stuck with vsh for now.
```

Comment:
This comment isn't accurate. You're not using vsh and you're setting this cli regardless.